sdk/azcore/testdata/perf/client_get.go

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package main
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/internal/pipeline"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/runtime"
+	"github.com/Azure/azure-sdk-for-go/sdk/internal/perf"
+)
+
+type clientGETTestOptions struct {
+	url string
+}
+
+var clientGetOpts clientGETTestOptions = clientGETTestOptions{url: ""}
+
+// sleepTestRegister is called once per process
+func clientTestRegister() {
+	flag.StringVar(&clientGetOpts.url, "url", "", "URL to send a GET request")
```

Comment:
What do you typically use here now? (maybe the bing.com default was never actually triggered in practice/)

Reply:
When I'm testing for validation I have used bing.com. I'm not sure what @mikeharder uses in the real performance tests.

Reply:
We currently do not run an HttpClient GET test in the perf automation.  If I were to run an HttpClient GET test manually, I would use a super-fast "hello world" web server (like https://github.com/mikeharder/MinKestrel/tree/master/MinKestrel) running on a VM in the same virtual network as the client VM.